modules/ROOT/pages/errors/gql-errors/22N60.adoc

```diff
@@ -0,0 +1,11 @@
+= 22N60
+
+== Status Description
+error: data exception - illegal element. Encountered illegal `{ <<item>> }` element: `{ <<msg>> }`.
+
```

Comment:
Hey @bigmontz, could you also provide an example here? You can take inspiration from 22N69, 22N71, etc.